homeassistant/components/awair/sensor.py

```diff
@@ -201,6 +206,7 @@ def extra_state_attributes(self) -> dict:
 
         https://docs.developer.getawair.com/?version=latest#awair-score-and-index
         """
+        assert self._air_data
```

Comment:
Not sure if this is OK, or if `self._air_data` can be `None` and that should be handled.

Reply:
See other comment.